plugins/org.komodo.core/src/org/komodo/repository/RepositoryImpl.java

```diff
@@ -356,6 +348,18 @@ public RepositoryImpl( final Type type,
         this.id = id;
     }
 
+    private UnitOfWork verifyTransaction(final UnitOfWork uow, String name, boolean rollback) throws KException {
```

Comment:
Possibly, except that can have a radically different message and message parameters, increasing the method signature.